rosetta/offline_config_mainnet.toml

```diff
@@ -0,0 +1,5 @@
+# OfflineSettings section of the rosetta server offline mod
+[OfflineSettings]
+    ChainID = "1"
+    MinGasPrice = 50000
+    MinGasLimit = 1000000000
```

Comment:
should be switched (min gas limit 50k, min gas price 1000000000)
+ empty line

Reply:
done.